pipelines/skylab/multiome/test_inputs/Scientific/10k_pbmc.json

```diff
@@ -0,0 +1,37 @@
+{
+"Multiome.annotations_gtf":"gs://broad-gotc-test-storage/Multiome/input/scientific/10k_PBMC_Multiome/introns_modified_gencode.vGencode_V43.primary_assembly.annotation.gtf",
+  "Multiome.barcodes_in_read_name":"true",
+  "Multiome.i1_fastq":[
```

Comment:
The JSON will need to be updated to match the latest WDL in beta-pipeline folder.